cmd/fortifyExecuteScan.go

```diff
@@ -606,9 +606,14 @@ func populatePipTranslate(config *fortifyExecuteScanOptions, classpath string) (
 	var translateList []map[string]interface{}
 	translateList = append(translateList, make(map[string]interface{}))
 
-	translateList[0]["pythonPath"] = classpath + ";" + config.PythonAdditionalPath
-	translateList[0]["pythonIncludes"] = config.PythonIncludes
-	translateList[0]["pythonExcludes"] = strings.ReplaceAll(config.PythonExcludes, "-exclude ", "")
+	separator := getSeparator()
+
+	translateList[0]["pythonPath"] = classpath + separator +
+		getSuppliedOrDefaultListAsString(config.PythonAdditionalPath, []string{}, separator)
+	translateList[0]["pythonIncludes"] = getSuppliedOrDefaultListAsString(
```

Comment:
Instead of `pythonIncludes` you could also name the key src. Same for `pythonExcludes`.